Core/Graphics/CMakeLists.txt

```diff
@@ -64,22 +70,31 @@ target_compile_definitions(Graphics
 set_property(TARGET Graphics PROPERTY FOLDER Core)
 source_group(TREE ${CMAKE_CURRENT_SOURCE_DIR} FILES ${SOURCES})
 
-add_library(GraphicsInternal INTERFACE)
-target_include_directories(GraphicsInternal
-    INTERFACE "Source"
-    INTERFACE "Source/${BABYLON_NATIVE_PLATFORM}")
-target_link_to_dependencies(GraphicsInternal
+if(APPLE)
+    # compile as objc++
+    target_compile_options(Graphics PUBLIC "SHELL:-x objective-c++")
+endif()
+
+add_library(GraphicsDevice INTERFACE)
+target_include_directories(GraphicsDevice
+    INTERFACE "Include/Shared"
+    INTERFACE "Include/Platform/${PLATFORM_PATH}"
+    INTERFACE "Include/RendererType/${GRAPHICS_API}")
+target_link_to_dependencies(GraphicsDevice
+    INTERFACE Graphics
+    INTERFACE JsRuntime)
+
+add_library(GraphicsDeviceContext INTERFACE)
+target_include_directories(GraphicsDeviceContext
+    INTERFACE "InternalInclude"
+    INTERFACE "InternalInclude/${BABYLON_NATIVE_PLATFORM}")
+target_link_to_dependencies(GraphicsDeviceContext
     INTERFACE Graphics
     INTERFACE JsRuntimeInternal
     INTERFACE arcana
     INTERFACE bgfx
     INTERFACE bimg
     INTERFACE bx)
 
-target_compile_definitions(GraphicsInternal
+target_compile_definitions(GraphicsDeviceContext
     INTERFACE NOMINMAX)
-
-if(APPLE)
-    # compile as objc++
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -x objective-c++")
-endif()
```

Comment:
What happened to this?

Reply:
This was not required, it was copied from the Graphics library and is now removed.